app/src/androidTest/java/com/github/onlynotesswent/ui/MainActivityTest.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.onlynotesswent.ui
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.onlynotesswent.MainActivity
+import junit.framework.TestCase.assertNotNull
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MainActivityTest {
```

Comment:
More thorough tests for activity should be added in the future as it is not sufficient to just verify the activity is not null.